grader/settings.py

```diff
@@ -22,16 +22,18 @@
 
 INSTALLED_APPS = (
     # 'django.contrib.admin',
-    # 'django.contrib.auth',
-    # 'django.contrib.contenttypes',
+    'django.contrib.auth',
```

Comment:
What requires this?

Reply:
Previously I also used django rest framework to build the api and it needs these. I removed the api using drf later but forgot to modify here. Now I comment them.

Reply:
> What requires this?

Previously I also used django rest framework to build the api and it needs these. I removed the api using drf later but forgot to modify here. Now I comment them.